external/test-runtime/XUnit.Runtime.depproj

```diff
@@ -7,6 +7,7 @@
     -->
     <NugetRuntimeIdentifier>win7-x64</NugetRuntimeIdentifier>
     <OutputPath>$(RuntimePath)</OutputPath>
+    <OutputPath Condition="'$(BinPlaceNETCoreAppTestDependencies)' == 'true'">$(NETCoreAppPackageRuntimePath)\..\dependencies</OutputPath>
```

Comment:
No, it is just my preference (to make it additive) as it is usually easy to combine two separated components into one, but the opposite is hard and error-prune.

One another reason is.. that I currently do not know how to use bin-placing logic.